planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -91,6 +102,7 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         adpYaml = DamGenerator.addApplicationInfo(adpYaml, slaInfoResponse, SLA_INFO_GROUPNAME);
 
         Map groups = (Map) adpYaml.remove(GROUPS);
+        adpYaml.remove(NODE_TYPES);
```

Comment:
Good question, @andreaturli.
I am not in charge of DamGenerator, but I will try to explain.
ADPs contain NodeTypes of SeaClouds which are used during AAM generation and ADP management. However, this type will be used by the Deployer. Current DamGenerator builds the DAM over the ADP object so the ADP's NodeTypes has to be removed. 
Of course, the best option it is refactor the current DamGenerator, and I think it must be done.